pkg/consts/consts.go

```diff
@@ -287,6 +287,15 @@ const (
 	CannotDeletePublicIPErrorMessageCode = "PublicIPAddressCannotBeDeleted"
 	// ReferencedResourceNotProvisionedMessageCode means the referenced resource has not been provisioned
 	ReferencedResourceNotProvisionedMessageCode = "ReferencedResourceNotProvisioned"
+
+	// LBInUseRawError is the LoadBalancerInUseByVirtualMachineScaleSet raw error
+	LBInUseRawError = `Retriable: false, RetryAfter: 0s, HTTPStatusCode: 400, RawError: Retriable: false, RetryAfter: 0s, HTTPStatusCode: 400, RawError: {
```

Comment:
move to the test file

Reply:
I put it into `azure_error.go` because there are unit tests from two different packages using it.